app/plugins/hmac-authentication.md

```diff
@@ -129,6 +129,12 @@ the order they appear.
 
 3. If value is not the last value then append an ASCII newline `\n`.
   The string MUST NOT include a trailing ASCII newline.
+  
+### Clock Skew
+
+Plugin also implements the clock-skew to prevent Replay Attack. The server and 
+requesting machine should be synchronized with NTP and a valid date in GMT format should 
+be sent using either `X-Date` or `Date` header.
```

Comment:
you can also implement `clock-skew` from the specification to prevent replay attacks. 

The server and requesting machine should be synchronized with NTP and a valid date in GMT format should be sent using either `X-Date` or `Date` headers.